public/docs/ts/latest/guide/architecture.jade

```diff
@@ -193,196 +221,214 @@ figure
-  
-  A directive is a class with directive metadata. In TypeScript we'd apply the `@Directive` decorator
+  Angular templates are *dynamic*. When Angular renders them, it transforms the DOM
+  according to the instructions given by **directives**.
+
+  A directive is a class with directive metadata. In !{_Lang} we apply the `@Directive` !{_decorator}
   to attach metadata to the class.
 <br clear="all">
 :marked
-  We already met one form of directive: the component. A component is a *directive-with-a-template*
-  and the `@Component` decorator is actually a `@Directive` decorator extended with template-oriented features.
+  We already met one form of directive: the component. A component is a *directive-with-a-template*;
+  a `@Component` !{_decorator} is actually a `@Directive` !{_decorator} extended with template-oriented features.
 
 .l-sub-section
   :marked
-    While the **component is technically a directive**, 
-    it is so distinctive and central to Angular applications that we chose
-    to separate the component from the directive in our architectural overview.
+    While **a component is technically a directive**,
+    components are so distinctive and central to Angular applications that we chose
+    to separate components from directives in this architectural overview.
 :marked
-  There are two *other* kinds of directives as well that we call "structural" and "attribute" directives.
-  
-  They tend to appear within an element tag like attributes, 
+  Two *other* kinds of directives exist: _structural_ and _attribute_ directives.
+
+  They tend to appear within an element tag like attributes,
   sometimes by name but more often as the target of an assignment or a binding.
-  
+
   **Structural** directives alter layout by adding, removing, and replacing elements in DOM.
-  
-  We see two built-in structural directives at play in our [example](#template) template:
-+makeExample('architecture/ts/app/hero-list.component.1.html', 'structural')(format=".")
+
+  Our [example](#templates) template uses two built-in structural directives:
+
++makeExcerpt('app/hero-list.component.1.html', 'structural')
+
 :marked
-  * [`*ngFor`](displaying-data.html#ngFor) tells Angular to stamp out one `<div>` per hero in the `heroes` list.
+  * [`*ngFor`](displaying-data.html#ngFor) tells Angular to stamp out one `<li>` per hero in the `heroes` list.
   * [`*ngIf`](displaying-data.html#ngIf) includes the `HeroDetail` component only if a selected hero exists.
-  
-  **Attribute** directives alter the appearance or behavior of an existing element. 
-  In templates they look like regular HTML attributes, hence the name.
-  
-  The `ngModel` directive, which implements two-way data binding, is an example of an attribute directive.
-+makeExample('architecture/ts/app/hero-detail.component.html', 'ngModel')(format=".")
+
+.l-sub-section
+  :marked
+    In Dart, **the only value that is true is the boolean value `true`**; all
```

Comment:
Perhaps this should be a DART DIFFERENCE in the Dart version only? I can't think of anywhere else where we have Dart info in the TS docs.